nilearn/image/image.py

```diff
@@ -1033,6 +1042,19 @@ def math_img(formula, **imgs):
      >>> result_img = math_img("img1 + img2",
      ...                       img1=anatomical_image, img2=log_img)
 
+    The result image will have the same shape and affine as the input images;
+    but might have different header information, specifically the TR value,
+    see :gh:`2645`.
+
+    .. versionadded:: 0.10.4.dev
+
+    We can also copy the header from one of the input images using
+    ``copy_header_from``::
+
+     >>> result_img_with_header = math_img("img1 + img2",
+     ...                                   img1=anatomical_image, img2=log_img,
+     ...                                   copy_header_from="img1")
```

Comment:
curious to see if doc test will work with this or if it will complain that `anatomical_image` is unknown for example

Reply:
Just checked, works fine. `anatomical_image` is defined just before for the pre-existing doc string example. It doesn't show up in the diff I think.

Reply:
ah!!! nice!!! did not realize this.